frontend/src/components/FormInput.tsx

```diff
@@ -26,7 +26,7 @@ const FormInput = ({
         errorMessage
           ? 'border border-red-500 text-red-900 focus:ring-red-500 focus:border-red-500'
           : '',
-        className,
+        className ? className : '',
```

Comment:
I don't understand this change. Please explain

Reply:
Primesc eroare daca nu fac verificarea asta: tip `string | undefined` nu poate fi asignat la parametru de tip `string`.

Reply:
in this case i think you can do className || ''